csi/crypto/crypto.go

```diff
@@ -72,8 +75,20 @@ func VolumeMapper(volume string) string {
 
 // EncryptVolume encrypts provided device with LUKS.
 func EncryptVolume(devicePath, passphrase string, cryptoParams *EncryptParams) error {
+	namespaces := []lhtypes.Namespace{lhtypes.NamespaceMnt, lhtypes.NamespaceIpc}
```

Comment:
I would suggest creating a few namespace groups for our default usage in common-libs and exporting them, so we need to redeclare these namespaces again. Like which cases require mnt and ipc or even net. 

Create a ticket for that to improve. This will make our code better maintainable.

Reply:
https://github.com/longhorn/longhorn/issues/7406